examples/dsf-quickstart/stacks/main.py

```diff
@@ -1,34 +1,49 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-import os
-from aws_cdk import Stack, RemovalPolicy
+from aws_cdk import Stack, RemovalPolicy, CfnOutput
+from aws_cdk.aws_iam import Policy, PolicyStatement
+from aws_cdk.aws_kms import Key
 from constructs import Construct
+from aws_cdk.aws_s3 import Bucket
 import aws_dsf as dsf
 
 
 class DataStack(Stack):
   def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
     super().__init__(scope, construct_id, **kwargs)
 
-    storage = dsf.storage.DataLakeStorage(
-            self, "DataLakeStorage", removal_policy=RemovalPolicy.DESTROY
-        )
-
-    catalog = dsf.governance.DataLakeCatalog(
-            self, "DataLakeCatalog",
-            data_lake_storage=storage,
-            database_name='spark_data_lake',
+    storage = dsf.storage.AnalyticsBucket(
+            self, "DataLakeStorage", 
             removal_policy=RemovalPolicy.DESTROY,
+            encryption_key= Key(self, "StorageEncryptionKey",
+                removal_policy=RemovalPolicy.DESTROY,
+                enable_key_rotation=True
+            )
         )
-    
+  
     # Use DSF on AWS to create Spark EMR serverless runtime, package Spark app, and create a Spark job.
     spark_runtime = dsf.processing.SparkEmrServerlessRuntime(
             self, "SparkProcessingRuntime", name="TaxiAggregation",
             removal_policy=RemovalPolicy.DESTROY,
         )
+    
```

Comment:
Can you add comment to explain why it's needed? I assume it's for reading from the source...